apps-rendering/src/renderer.ts

```diff
@@ -177,13 +177,15 @@ const plainTextElement = (node: Node, key: number): ReactNode => {
 const shouldShowDropCap = (
```

Comment:
I guess you saw this but icymi I noticed DCR has it's own version of this function:

https://github.com/guardian/dotcom-rendering/blob/9fcc548a54cecc6a8c17da8454a90b78487a0409/dotcom-rendering/src/components/TextBlockComponent.tsx#L106-L128

Which leads to a handy `isDinkus()` and what looks like dinkus variations (ha!):

https://github.com/guardian/dotcom-rendering/blob/9fcc548a54cecc6a8c17da8454a90b78487a0409/dotcom-rendering/src/model/enhance-dividers.ts#L4-L22

Reply:
Yes I did, but AR has always done its own thing so I don't want to change that now :(